tests/PHPUnit/System/expected/test_CustomEvents_Events.getAction_flat__API.getProcessedReport_day.xml

```diff
@@ -43,16 +43,7 @@
 	<reportData>
 		<result prettyDate="Sunday 3 January 2010">
 			<row>
-				<label>playTrailer - Ponyo (崖の上のポニョ)</label>
```

Comment:
Here we currently sort by nb_visits by default which is not present in flattened somehow but in the normal report